packages/terra-file-path/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-file-path",
+  "main": "lib/FilePath.js",
+  "version": "0.1.0",
+  "description": "Terra File Path is a horizontal menu that indicates where the user is. It is also more commonly known as a breadcrumb component.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git",
+    "directory": "packages/terra-file-path"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-file-path",
+    "file-path",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": ">=2.8.0, <6.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "focus-trap-react": "^6.0.0",
+    "keycode-js": "^3.1.0",
+    "prop-types": "^15.5.8"
+  },
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
```

Comment:
can we add a `lint:js:fix` script?

Reply:
[b8a0750](https://github.com/cerner/terra-framework/pull/1771/commits/b8a075007e53efdc47e7cfb0034d0ac7cb89bd5f)